tests/unit/test_charm.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python3
+"""
```

Comment:
Why is this not a single line comment instead of a block?

Reply:
It's either me or `black`, but I bet it's me, will fix it :) Thanks!